bkmeans/bkmeans.py

```diff
@@ -47,18 +48,17 @@ def get_utility(self, X, C):
         # aggregate utility for each centroid
         return np.array([np.sum(util[dist_srt[:, 0]==i]) for i in range(len(C))])
 
-    def _lloyd(self,C,X):
+    def _lloyd(self,C,X,sample_weight):
         """perform Lloyd's algorithm"""
         self.init = C # set cluster centers
         self.n_clusters = len(C) # set k-value
-        super().fit(X) # Lloyd's algorithm, sets self.inertia_ (a.k.a. phi)
+        super().fit(X=X, sample_weight=sample_weight) # Lloyd's algorithm, sets self.inertia_ (a.k.a. phi)
 
     def fit(self, X, sample_weight=None):
```

Comment:
is default of None still applicable here?

Reply:
I see no reason why it shouldn't